src/nyc_trees/js/src/surveyPage.js

```diff
@@ -110,7 +110,9 @@ var dom = {
 
     formTemplate = Handlebars.compile($(dom.treeFormTemplate).html()),
 
-    blockfaceId = mapUtil.getBlockfaceIdFromUrl(),
+    // jshint complains when initializing this to undefined
+    blockfaceId = -1,
```

Comment:
Consider just declaring this but leaving it undefined, to prevent the lint, instead of assigning `-1`:
```javascript
var ... = ...,
     blockface,
     ... = ...,
```